core/commands/commit.py

```diff
@@ -44,7 +45,14 @@ def run_async(self, repo_path=None, include_unstaged=False, amend=False):
         view.settings().set("git_savvy.commit_view.include_unstaged", include_unstaged)
         view.settings().set("git_savvy.commit_view.amend", amend)
         view.settings().set("git_savvy.repo_path", repo_path)
-        view.set_syntax_file("Packages/GitSavvy/syntax/make_commit.tmLanguage")
+
+        gitsavvy_settings = sublime.load_settings("GitSavvy.sublime-settings")
+        if gitsavvy_settings.get("use_syntax_for_commit_editmsg")
+            syntax_file = util.file.get_syntax_for_file("COMMIT_EDITMSG")
+            view.set_syntax_file(syntax_file)
+        else:
+            view.set_syntax_file("Packages/GitSavvy/syntax/make_commit.tmLanguage")
+
```

Comment:
Overriding the default syntax is a good approach to resolve #41.  However, let's simplify by creating a setting in `GitSavvy.sublime-settings` called `commit_syntax` with a default value of `Packages/GitSavvy/syntax/make_commit.tmLanguage` (and remove `use_syntax_for_commit_editmsg`).  Then, replace the above chunk of code with:
```python
gitsavvy_settings = sublime.load_settings("GitSavvy.sublime-settigns")
view.set_syntax_file(gitsavvy_settings.get("commit_syntax")
```